src/tests/quantuminspire/test_api.py

```diff
@@ -1004,6 +1004,16 @@ def test_wait_for_completed_job_returns_false(self):
         is_completed = api._wait_for_completed_job(quantum_inspire_job, collect_max_tries, sec_retry_delay=0.0)
         self.assertFalse(is_completed)
 
+    def test_wait_for_cancelled_job_returns_false(self):
+        job_id = 509
+        expected_payload = {'id': job_id}
+        self.coreapi_client.handlers['jobs'] = partial(self.__mock_job_handler, expected_payload, 'read',
+                                                       status='CANCELLED')
+        api = QuantumInspireAPI('FakeURL', self.authentication, coreapi_client_class=self.coreapi_client)
+        quantum_inspire_job = QuantumInspireJob(api, job_id)
+        is_completed = api._wait_for_completed_job(quantum_inspire_job)
```

Comment:
Not a big fan of calling private methods but I see it is done a lot in this test class.